src/mumble/ServerHandler.cpp

```diff
@@ -22,6 +28,31 @@
 #include "HostAddress.h"
 #include "ServerResolver.h"
 #include "ServerResolverRecord.h"
+#include "Utils.h"
+
+#include <QtCore/QtEndian>
+#include <QtGui/QImageReader>
+#include <QtNetwork/QSslConfiguration>
+#include <QtNetwork/QUdpSocket>
+
+#include <openssl/crypto.h>
+
+#ifdef Q_OS_WIN
+// This header is not protected with an include guard on MinGW,
+// which results in redefinitions if the PCH header is used.
```

Comment:
…but we skip the import of a define of it is already defined thus working around this issue? The comment should make that clear, that this is a workaround for that. Not that that is an open issue that occurs here.